src/components/Graph/Graph.tsx

```diff
@@ -15,15 +15,20 @@ type GraphProps = {
 }
 
 export const Graph = ({ asset, timeframe, setPercentChange, isLoaded }: GraphProps) => {
-  const [data, setData] = useState<HistoryData[]>([])
+  const [data, setData] = useState<HistoryData[] | null>([])
```

Comment:
when does this get set to null?

Reply:
we changed the api of market service to return null on network failure rather than an empty array

it makes it easier to check than checking for array lengths

in reality no one will actually call `setData(null)` it's just to satisfy the compiler